README.md

```diff
@@ -180,16 +180,69 @@ The following is an example configuration when you are adding an `IdentifiedData
 
 If you want to add a `Portable` class, you should use `<portable-factories>` instead of `<data-serializable-factories>` in the above configuration.
 
-#### 1.2.4. Using hazelcast-member Tool
+#### 1.2.1.3. Using hazelcast-member Tool
 
-`hazelcast-member` is a tool to download and run Hazelcast IMDG members easily. If you have brew installed, run the following commands to instal this tool:
+`hazelcast-member` is a tool to download and run Hazelcast IMDG members easily. 
+
+You can find the installation guides for various platforms below.
+
+##### Installation Guide for Mac OS X
+
+If you have brew installed, run the following commands to install this tool:
 
 ```
 brew tap hazelcast/homebrew-hazelcast
 brew install hazelcast-member
 ```
 
-Now, you can start a member by running the following command:
+##### Installation Guide for Ubuntu and Debian
+
+You can resolve the `.deb` artifacts from Bintray as follows:
+
+First, you need to import the Bintray's GPG key:
+
+```
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
+```
+
+Then, add the following to your `/etc/apt/sources.list` system config file and update the lists of packages:
```

Comment:
Then, run the following commands to add the `.deb` artifact to your system configuration file and update the lists of packages: